Supertext.Base.Net/Mail/MailService.cs

```diff
@@ -13,11 +15,13 @@ namespace Supertext.Base.Net.Mail
     internal class MailService : IMailService
     {
         private static ILogger<IMailService> _logger;
-        private readonly MailServiceConfig _configuration;
+        private readonly MailServiceConfig _mailServiceConfig;
+        private readonly IConfiguration _configuration;
 
-        public MailService(ILogger<IMailService> logger, MailServiceConfig configuration)
+        public MailService(ILogger<IMailService> logger, MailServiceConfig mailServiceConfig, IConfiguration configuration)
```

Comment:
If you're using the `configuration` object to obtain "Aspose-EmailLicense" then why not add this as a property on the `MailServiceConfig` class?